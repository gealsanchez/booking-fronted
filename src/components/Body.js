/* eslint-disable jsx-quotes */
import React from 'react';
import { Link } from 'react-router-dom';

const Body = () => (
  <header className="header">
    <span className="header-title">Book a ticket with your prefer automobile</span>
    <div className="header-automobile-btn-space">
      <span className="header-automobile-btn">
        <Link to="/signin" className="link-sign">Sign In</Link>
      </span>
      <span className="space">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;</span>
      <span className="header-automobile-btn-signup">
        <Link to="/signup" className="link-sign">Sign Up</Link>
      </span>
    </div>
  </header>
);

export default Body;
