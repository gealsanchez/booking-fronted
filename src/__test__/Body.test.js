import React from 'react';
import { Provider } from 'react-redux';
import { render } from '@testing-library/react';
import { BrowserRouter as Router } from 'react-router-dom';
import store from '../redux/configureStore';
import Body from '../components/Body';

it('Body page should render correctly', () => {
  const result = render(
    <React.StrictMode>
      <Router>
        <Provider store={store}>
          <Body />
        </Provider>
      </Router>
    </React.StrictMode>,
  );
  expect(result).toMatchSnapshot();
});
