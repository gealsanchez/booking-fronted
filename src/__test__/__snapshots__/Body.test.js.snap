// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Body page should render correctly 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <header
        class="header"
      >
        <span
          class="header-title"
        >
          Book a tcket with your prefer automobile
        </span>
        <div
          class="header-automobile-btn-space"
        >
          <span
            class="header-automobile-btn"
          >
            <a
              class="link-sign"
              href="/signin"
            >
              Sign In
            </a>
          </span>
          <span
            class="space"
          >
                  
          </span>
          <span
            class="header-automobile-btn-signup"
          >
            <a
              class="link-sign"
              href="/signup"
            >
              Sign Up
            </a>
          </span>
        </div>
      </header>
    </div>
  </body>,
  "container": <div>
    <header
      class="header"
    >
      <span
        class="header-title"
      >
        Book a tcket with your prefer automobile
      </span>
      <div
        class="header-automobile-btn-space"
      >
        <span
          class="header-automobile-btn"
        >
          <a
            class="link-sign"
            href="/signin"
          >
            Sign In
          </a>
        </span>
        <span
          class="space"
        >
                
        </span>
        <span
          class="header-automobile-btn-signup"
        >
          <a
            class="link-sign"
            href="/signup"
          >
            Sign Up
          </a>
        </span>
      </div>
    </header>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
